components/Carousel/Carousel.js

```diff
@@ -1,8 +1,81 @@
+let current;
+
+const setDispToNone = () => {
+  carousel.lastPic.style.display = "none";
```

Comment:
I see you have a great understanding of the DRY principle! 

https://en.wikipedia.org/wiki/Don't_repeat_yourself